.editorconfig

```
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Testing the new suggestion feature

```suggestion
insert_final_newline = false

```

---

Handy. I have a big apply suggestion button. Although I think you meant to change it to true.

---

Ah! Yes!